mypyc/primitives/bytes_ops.py

```diff
@@ -1,11 +1,28 @@
 """Primitive bytes ops."""
 
-from mypyc.ir.rtypes import object_rprimitive
-from mypyc.primitives.registry import load_address_op
+from mypyc.ir.ops import ERR_MAGIC
+from mypyc.ir.rtypes import object_rprimitive, bytes_rprimitive
+from mypyc.primitives.registry import load_address_op, function_op
 
 
 # Get the 'bytes' type object.
 load_address_op(
     name='builtins.bytes',
     type=object_rprimitive,
     src='PyBytes_Type')
+
+# bytes(obj)
+function_op(
+    name='builtins.bytes',
+    arg_types=[object_rprimitive],
+    return_type=bytes_rprimitive,
+    c_function_name='PyBytes_FromObject',
+    error_kind=ERR_MAGIC)
+
+# bytearray(obj)
+function_op(
```

Comment:
Add also run tests to make sure the C API functions do what they should. For example, test `bytes([5])`, `bytes(bytearray(b'foo'))` and `bytes('x')` (the last one should raise an exception).